tools/src/test/java/io/github/ygojson/tools/yugipedia/api/YugipediaApiIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+package io.github.ygojson.tools.yugipedia.api;
+
+import java.io.IOException;
+
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import retrofit2.Response;
+
+import io.github.ygojson.tools.yugipedia.api.params.Category;
+import io.github.ygojson.tools.yugipedia.api.params.PipeSeparated;
+import io.github.ygojson.tools.yugipedia.api.params.SortDirection;
+import io.github.ygojson.tools.yugipedia.api.response.QueryResponse;
+
+@Slf4j
+class YugipediaApiIntegrationTest {
```

Comment:
Require some tests that are not integration against the production database. Create some json mocks from a couple of server requests and use them on a mock server to ensure that response/params changes do not break the contract (and being able to compare them if something changes on the api.

Reply:
Also maybe we should extrac the integration tests to a separate place.